Dockerfile

```diff
@@ -8,6 +8,8 @@ ARG PYPERF_BUILDER_UBUNTU=@sha256:cf31af331f38d1d7158470e095b132acd126a7180a54f2
 ARG PERF_BUILDER_UBUNTU=@sha256:d7bb0589725587f2f67d0340edb81fd1fcba6c5f38166639cf2a252c939aa30c
 # phpspy - ubuntu:20.04
 ARG PHPSPY_BUILDER_UBUNTU=@sha256:cf31af331f38d1d7158470e095b132acd126a7180a54f263d386da88eb681d93
+# dotnet builder ubuntu:20.04
```

Comment:
It's not really `ubuntu:20.04` - the image is `mcr.microsoft.com/dotnet/sdk` and what tag did you use to get the hash from?

Reply:
`docker inspect --format='{{index .RepoDigests 0}}' mcr.microsoft.com/dotnet/sdk:6.0-focal
mcr.microsoft.com/dotnet/sdk@sha256:749439ff7a431ab4bc38d43cea453dff9ae1ed89a707c318b5082f9b2b25fa22`

Reply:
Cool - can you please modify the comment with that tag? So we know how to update it in the future.